components/js/slickGrid.ts

```diff
@@ -533,16 +539,8 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
     }
 
     private renderGridDataRowsRange(startIndex: number, count: number): void {
-        let editor = this._grid.getCellEditor();
-        let oldValue = editor ? editor.getValue() : undefined;
-        let wasValueChanged = editor ? editor.isValueChanged() : false;
         this.invalidateRange(startIndex, startIndex + count);
-        let activeCell = this.activeCell;
```

Comment:
scenario: I edited a cell  and scroll away without moving the focus, when I scroll back my edits will be gone. this doesn't seem correct to me.

anyone remember why was this needed?

Reply:
This looks like it only applies when you edit a cell that that is invalidated, this is trying to reapply the edit when slickgrid will discard that edit.

Reply:
Not sure you need to remove the code.

Reply:
@anthonydresser Based on what I observed from the following scenario, i think the logic is wrong, I am not in office today, could you please give it a try?
1. open edit data window for a table with more than 1000 rows (e.g. HumanResources.Employee table in adventureworks database)
2. select 1000 in the max rows drop down in the toolbar and click run button
3. edit a cell in the first row and leave the focus inside the text edit
4. scroll all the way down to the end of grid and then scroll back to the top
Expected: the updated value should still be there
Actual: the change is reverted

Reply:
I see the scenario you are talking about, but this code does not impact that scenario. What you are talking about is a separate thing. 

What this code is doing is if the user is editing row 99 and for some reason (this code was written awhile ago by another team) the async data provider attempts to invalidate the rows 50-100 (perhaps on slow servers?) this code attempts to do that invalidation then reapply the edit to row 99.

Reply:
I think you are right, the issue is a separate thing, I tested again, removing it doesn't actually help, I didn't scroll away too far that's why i was under the impression that this code is causing the problem.

but this code seems to be doing nothing, it is simply getting the current value (getValue()) from the editor and reapply it.